be/src/util/core_local.cpp

```diff
@@ -61,6 +65,7 @@ class CoreDataAllocatorImpl : public CoreDataAllocator {
     }
 private:
     static constexpr int ELEMENTS_PER_BLOCK = BLOCK_SIZE / ELEMENT_BYTES;
+    SpinLock _lock; // lock to protect the modification of _blocks
```

Comment:
CoreData is to bind data to cpu to eliminate the lock.
If add a lock, it's design is meaningless.

Reply:
This is only used when creating the core local value (by calling `get_or_create()` of `CoreDataAllocatorImpl`).
And when creating the value, there is nothing to do with "cpu-bind".

The so called "cpu-bind" operation happens when accessing the `CoreLocalValue`,
not related to `CoreDataAllocatorImpl`

Reply:
OK, I have no problem.